usage.js

```diff
@@ -34,6 +34,16 @@ const acc = new Auth({
 
 const sdk = new SDK(acc);
 
+const ipfs = new IPFS({
```

Comment:
this doesn't work: where does IPFS is defined?

You're missing the export in index.js i will say :)

Reply:
BTW, we should integrate the IPFS with Auth class, to have an "all-in-one" definition of the credentials parameters.

Reply:
```
const acc = new Auth({
  privateKey: process.env.WALLET_PRIVATE_KEY,
  projectId: process.env.INFURA_PROJECT_ID,
  secretId: process.env.INFURA_PROJECT_SECRET,
  rpcUrl: process.env.EVM_RPC_URL,
  chainId: 5,
  ipfsUrl: process.env.INFURA_IPFS_ENDPOINT,
});

const sdk = new SDK(acc);
const hash = sdk.ipfs.uploadFile({
  source: './integration-test/ipfs-test/infura.png',
});
```

WDYT?

Reply:
yes you're right, i have to export in index.js, for the integration of IPFS module in the same class, we're following the recommandation of adrien delaroche for now , that why it's a separated module

Reply:
IPFS is not under web3 project so it's another pair of credentials

Reply:
even with another set of credentials, it's our kitchen to manage. We don't want the dev user to bother with increased complexity. We can do something like:
```
const acc = new Auth({
  privateKey: process.env.WALLET_PRIVATE_KEY,
  projectId: process.env.INFURA_PROJECT_ID,
  secretId: process.env.INFURA_PROJECT_SECRET,
  rpcUrl: process.env.EVM_RPC_URL,
  chainId: 5,
  ipfs: {
    url: process.env.INFURA_IPFS_ENDPOINT,
    projectId: process.env.INFURA_IPFS_PROJECT_ID,
    secretId: process.env.INFURA_IPFS_SECRET_ID,
  }
});
```
WDYT?